src/BaseBuilder.js

```diff
@@ -117,6 +118,22 @@ class BaseBuilder {
 	}
 
 	/**
+	* @description Add functions for function_score queries
+	* @param {string|Object} field - Field/type for sorting
+	* @param {string|Object} value - Value of the Field/Type
+	* @return {BaseBuilder} this
+	*/
+	func (field, value) {
```

Comment:
So field can actually be the field or a query object.  Should it still be called field?